README.md

```diff
@@ -1,2 +1,23 @@
-# HW01
-Edit the readme
+---
+title: "Homework 1"
+author: "Ben Wang"
+---
+
+### bwang5@uchicago.edu
+
+Hello all!
+My name is Ben, I'm an incoming first year student in the **University of Chicago** *Neurobiology* program [link here](https://neuroscience.uchicago.edu/). I am interested in studying the molecular mechanisms behind automatic functions such as breathing, pain and prioprioception. I have no prior experience in coding but have always been interested in learning!
+
+I am originally from Winnipeg, Canada, and came to the US for college. I majored in molecular and cellular biology and minored in music.
```

Comment:
So cool that you minored in music!  Although I don't have a formal minor in theatre, I spent so much time working on productions you could almost say I minored in theatre.

I also grew up playing the piano!

Reply:
It's awesome that both of you are involved in some kind of arts. I used to write a lot and drew a bit. I also took up piano for two/three years when I was a kid but due to lack of practice, I forgot everything! :( I really should try getting back into all these, I even got myself a keyboard so that I can start practicing the piano again.

Reply:
Nabilah, Ariana and I are now totally coming to visit you so we can use your keyboard hahaha. I know UChicago has some practice rooms available in the music building that are open to music students. I'm hoping that we can apply for a pass to use the pianos there as well!